binscripts/rvm-installer

```diff
@@ -418,7 +418,7 @@ verify_package_pgp()
     log "\
 GPG signature verification failed for '$1' - '$3'! Try to install GPG v2 and then fetch the public key:
 
-    ${SUDO_USER:+sudo }${rvm_gpg_command##*/} --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+    ${SUDO_USER:+sudo }${rvm_gpg_command##*/} --keyserver hkp://keys.openpgp.org --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
@mavaddat have you verified that keys.openpgp.org is working for you? I found I was only able to succeed with `hkp://pgp.mit.edu`. I can't recall if this is good practice or not, but would testing a :success response from the relevant server with the keys be a smart thing to add to the test suite? Additionally, might moving both server and keys to a config file reduce a bit of heartburn later?

Reply:
actually `keys.openpgp.org` is working for me as well, but not `pgp.mit.edu`.
but for consistency, I think we should stick with `keyserver.ubuntu.com` as I don't like having to change it at several places again... but I do prefer `keys.openpgp.org` actually...